OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -0,0 +1,97 @@
+Apache Web Server with Oracle WebLogic Server Proxy Plugin on Docker
+===============
+This project includes a quick start Dockerfile and samples for standalone Apache Web Server with 12.2.1.3.0 Oracle WebLogic Server Proxy Plugin based on Oracle Linux and Oracle JDK 7 (Server). The certification of Apache on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize/tweak, or create from scratch new scripts and Dockerfiles.
+
+## How to Build Apache With Plugin Docker Image
+
+This project offers a Dockerfile for Apache Web Server with Oraacle WebLogic Server Proxy Plugin in standalone mode. To assist in building the images, you can use `buildDockerImage.sh` script. See below for instructions and usage.
+
+The `buildDockerImage.sh` script is just a utility shell script that performs MD5 checks and is an easy way for beginners to get started. Expert users are welcome to directly call docker build with their preferred set of parameters.
+
+IMPORTANT: You have to download the **Oracle WebLogic Server Proxy Plugin 12.2.1.3.0** package (see .download file) and drop them in this directory.
+
+Run `buildDockerImage.sh` script.
+
+        $ sh buildDockerImage.sh 
+
+## How to run container 
+
+Run an Apache container to access an admin server that is running on <host> and listening to <port>.
+
+        $ docker run -d -e WEBLOGIC_HOST=i<host> WEBLOGIC_PORT=<port> -p 80:80 12213-apache
+
+Run an Apache container to access and load balance to a list of managed servers running standalone 
+ 
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port --net=<some net> -p 80:80 12213-apache
+
+Run an Apache image to proxy and load balance to a list of managed servers in a cluster
+     
+        Use a list of hosts and ports.
+
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port --net=<some net> -p 80:80 12213-apache
+
+        Or use a cluster URL if it is available
+
+        $ docker run -d -e WEBLOGIC_CLUSTER=<cluster-url> --net=<some net> -p 80:80 12213-apache
+
+The values of **WEBLOGIC_CLUSTER** must be valid, existing containers running WebLogic servers.
+
+If you are using multihost network, remove `--link` and set `--net=<your net>`.
+
+### Admin Server Only Example
+ 
+First make sure you have the WebLogic Server 12.2.1.3 sample and build your own image 12213-domain at [https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-domain].
```

Comment:
For Admin Server only containers they either must pull the image from the Docker Store 'docker pull store/oracle/weblogic:12.2.1.3' or they can build it them selves from https://github.com/oracle/docker-images/tree/master/OracleWebLogic/dockerfiles/12.2.1.3.  The 12213-domain sample is meant to run a domain with a cluster.

Reply:
We are showing apache part for admin server only. It is okay to use a domain with clusters although we only use admin server in the example. We are going to add cluster examples in the near future.

Reply:
There are two WebLogic topologies we support in Docker, the single Admin server/domain per container which you get from running a container from the WebLogic install image, and the WebLogic domain/cluster with one admin server and multiple clustered managed servers.  Your example for the single Admin server should use the WebLogic install image to run an admin server container, the WLST that is used in that image creates a domain with a single admin server. The 12213-domain WLST scripts that get invoked when you run the Admin server container create a WLS domain with an admin server, cluster with multiple managed servers.